multi-node/aws/pkg/config/templates/cloud-config-controller

```diff
@@ -36,14 +36,31 @@ coreos:
       command: start
       enable: true
       content: |
+{{ if isset .ContainerRuntime "rkt" }}
+        [Unit]
+        Requires=load-rkt-stage1.service
+        After=load-rkt-stage1.service
+{{ end }}
         [Service]
         Environment=KUBELET_VERSION={{.K8sVer}}
         Environment=KUBELET_ACI={{.HyperkubeImageRepo}}
-        Environment="RKT_OPTS=--volume dns,kind=host,source=/etc/resolv.conf --mount volume=dns,target=/etc/resolv.conf"
+        Environment="RKT_OPTS=--volume dns,kind=host,source=/etc/resolv.conf \
+          --mount volume=dns,target=/etc/resolv.conf \
+          --volume rktbin,kind=host,source=/usr/bin/rkt \
+          --mount volume=rktbin,target=/usr/bin/rkt \
```

Comment:
Sooo, how do you feel about the idea of creating a bash script somewhere in the hyperkube container with the contents:

```sh
#!/bin/sh
exec nsenter -m -u -i -n -p -t 1 -- $0 $@
```

and then symlinking it as `/usr/bin/rkt` or whatever matches the exact path on the host?

This would specifically work around the issue of `rkt gc` by allowing the `var-lib-rkt` bindmount to not exist and the `gc` to be run on the host of things.
I recall suggesting it before, but now it seems like we have a concrete reason to reconsider it.